lib/taskana-core/src/test/java/pro/taskana/impl/integration/ClassificationServiceImplIntExplicitTest.java

```diff
@@ -123,17 +122,16 @@ public void testInsertClassification()
 
         // does exist already
         try {
-            expectedClassification = (ClassificationImpl) this.createNewClassificationWithUniqueKey();
+            expectedClassification = (ClassificationImpl) this.createNewClassificationWithUniqueKey(domain, "t1");
             expectedClassification.setKey(key);
```

Comment:
This line is reduncant, becasue Key+Domain are given line above.

Reply:
Not really. createNewClassificationWithUniqueKey creates -as the name implies- a classification with a unique key. The purpose of the test is to create a ClassificationAlreadyExistsException. To do this, it overrides the unique key that was created in method createNewClassificationWithUniqueKey by a duplicate key thats stored in variable key.